libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/location/MetricsLocation.kt

```diff
@@ -0,0 +1,18 @@
+package com.mapbox.services.android.navigation.v5.internal.location
+
+import android.location.Location
+
+class MetricsLocation(internal val location: Location?) {
```

Comment:
Why it is `internal` but not `private` ?

Reply:
It's accessed from `NavigationMetricsWrapper` methods.